depscan/cli.py

```diff
@@ -1077,6 +1088,17 @@ def main():
         else DEFAULT_TERMINAL_THEME,
     )
     utils.export_pdf(html_file, pdf_file)
+    # render report into template if wished
+    if args.report_template:
```

Comment:
@heubeck Let's slightly modify this logic. We can avoid the nested if statements because python will not try to check for the truth of `os.path.isfile(args.report_template) if the first condition, is false. We avoid outputting the warning inappropriately when the user isn't trying to generate a custom report by changing the else to an elif.

```python
    if args.report_template and os.path.isfile(args.report_template):
        utils.render_template_report(
            jsonl_report_file=report_file,
            summary=summary,
            template_file=args.report_template,
            result_file=os.path.join(reports_dir, args.report_name),
        )
    elif args.report_template:
        LOG.warning("Template file %s doesn't exist, custom report not created.", args.report_template)
```